server.ts

```diff
@@ -516,6 +526,31 @@ app.get("/allowlist", function (req, res) {
   res.json({ allowlist: parseHost(jupyterGlobusMap)});
 });
 
+/**
+ * @openapi
+ * /announcement:
+ *  get:
+ *      description: Returns list of current announcements (Authentication NOT REQUIRED)
+ *      responses:
+ *          200:
+ *              description: Returns array of current announcements
+ */
+app.get("/announcement", function (req, res) {
```

Comment:
In the future please do these PRs as separate branches. It seems this PR contains all of the commits from the announcement route PR.